package org.sas04225.proto;

option java_package = "org.sas04225.proto";
option java_outer_classname = "DescriptorSetBuilderResultProto";

message DescriptorSetBuilderResult
{
	required string group_name = 1;
	required uint32 descriptor_count = 2;
	required uint64 startIndex = 3;
	required uint64 endIndex = 4;
}
